### ISSUE TITLE
Should the balloon pop up when part of a link is selected?

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-link#143]
It came out during testing the unlink option: https://github.com/ckeditor/ckeditor5-link/pull/142#issuecomment-316923545.

![jul-21-2017 09-11-54](https://user-images.githubusercontent.com/156149/28452812-aaa43206-6df4-11e7-9aa1-5594e05a4e44.gif)

Since we're planning to remove the unlink button from the toolbar I think that the balloon should show up in more cases because it's not that intuitive that you need to press Ctrl+K or the *link* button to unlink part of a text.

**Comment by oleq** [comment]
What would happen if one changed the URL and saved it in such situation? Split the link? Update the entire link? This change could affect the UX of the linking in general.

**Comment by Reinmar** [comment]
Yeah. This part is tricky. You may really want to achieve both results – changing just a part of it or the whole link.

However, I'd say that the actions you'd want to do for partial selections are (in frequency order):

* unlink (!!!!!!!)
* change the whole link
* change the selected part of the link

Which seriously worries me since [we decided to remove the unlink button from the toolbar](https://github.com/ckeditor/ckeditor5-link/issues/52). We need the partial unlinking functionality somewhere. We also need the ["open link" feature](https://github.com/ckeditor/ckeditor5-link/issues/31), so maaaybe, we should think how to provide both in some simplified balloon? IDK.
